.circleci/test/assembly/windows-zip/README.md

```diff
@@ -0,0 +1,61 @@
+# Workbase CI on Windows
```

Comment:
And does the package name need to be `../windows-zip`? I.e. Do/Will we have tests for `../windows-<non-zip>`? If so, will they belong inside or outside of this package? 

Given that:
a) we don't have any other assembly tests for windows,
b) if we do they will go in the same package, and
c) you already name the python test script `windows-zip.py`, therefore...
I think this package should be called just `../windows`

Reply:
I agree we should call the folder `windows`. Should then the `py` file be called `zip.py`? eg., `test/assembly/windows/zip.py`?

Reply:
Fixed in f7c3ce3daa038fe4dc7aff7f125f620e64e3dfa9